pkg/policy/portmap.go

```diff
@@ -0,0 +1,164 @@
+// Copyright 2020 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package policy
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/cilium/cilium/pkg/iana"
+	"github.com/cilium/cilium/pkg/u8proto"
+)
+
+var (
+	ErrNilMap               = errors.New("Nil map")
+	ErrUnknownNamedPort     = errors.New("Unknown named port")
+	ErrIncompatibleProtocol = errors.New("Incompatible protocol")
+	ErrNamedPortIsZero      = errors.New("Named port is zero")
+	ErrDuplicateNamedPorts  = errors.New("duplicate named ports")
```

Comment:
Nit: I think the convention for error strings is to start lower-cased.

Reply:
Fixed, thanks!